charts/db-operator/README.md

```diff
@@ -1,6 +1,26 @@
 # db-operator
 DB Operator is Kubernetes operator
 
+## Upgrading to `v1.10.0`
+CRDs are moved to the `templates` folder, so now they are part of the release. It means that when migration you will get errors about resource ownership. Thow errors will contain messages about missing `labels` and `annotations`, and the easiest way to fix it, will be just to add the `metadata` that helm can't find. So you can follow those messages one by one and when all the `CRDs` are patched, you'll be able to install the release.
```

Comment:
We had this already in the past and was not well accepted, related for upgrade issues etc.

https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

would maybe then go to the seperated helm chart solution, if the `crd` folder is also not a good solution.

Reply:
I guess you were just too fast for others :)
I think that a separated chart would just add complexity to installation. To me, it looks like if CRDs would be installed within `db-instance` chart, they should be put to `./crd` or to another chart. But `db-operator` has no resources that depend on `CRDs`, so I think it's already a separated chart. Also, it seems like big players like argo and cert-manager are doing it that way, and personally, I'd say that it's easier to update them then `kube-prometheus`, that is not taking care of CRDs, or Istio that has a separated chart for CRDs only.

What were those upgrade issues?

Reply:
And also, you always can set `Values.crds.install: false` and manage CRDs manually

Reply:
I've found an issue where it was discussed https://github.com/kloeckner-i/db-operator/issues/118.  I don't mind putting it to a different chart, but I think I'm following @dabde arguments there more :) 

I think that this chart is already a separated one, and db-operator won't work without CRDs, so it seems that they should be a part of this chart, but with the whole webhook thing, `./crds` dir is not an option, because it can't be templated. If you user wants to use `db-operator` and manage crds manually, they can do it, but I don't think it's a good idea, because they will have to deal with certificates injections and webhook stuff.

Having a separate chart means we have two packages, and both of them are not self-sufficient, because CRDs are nothing without operator and vice-versa.